### ISSUE TITLE
In screen reader mode, users have to tab bw activity bar sections

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#200165]
1. enable screen reader mode
2. focus the activity bar
3. try to use the down arrow key to reach the bottom section of the activity bar
4. 🐛 users have to use tab or shift tab

**Comment by meganrogge** [comment]
@sandy081 do you know of any difference in how this works for screen reader mode that could explain this?

**Comment by rzhao271** [comment]
I can use the down key to reach the bottom, but then I can't use the up key to go back up; focus gets stuck on the Account icon.

**Comment by rzhao271** [comment]
Confirmed that I still see the issue on the latest Insiders on Windows 11.

![Up arrow and shift+tab still doesn't work when focused on accounts in activity bar](https://github.com/microsoft/vscode/assets/7199958/9d08c2c4-d58c-4128-a3c6-50df0d7dacf0)

**Comment by meganrogge** [comment]
Closing as I have seen no users reporting that this is an issue for them
